### ISSUE TITLE
[BUG] video is cropped in docker container

### ISSUE THREAD

**Issue by tking16** [microsoft/playwright#26688]
Are we able to re-open this ticket? As I'm seeing it happen on webkit desktop (Docker container only)

https://github.com/microsoft/playwright/issues/7653

Playwright version: Version 1.35.1

config file:
```
import { defineConfig, devices } from '@playwright/test';

/**
 * Read environment variables from file.
 * https://github.com/motdotla/dotenv
 */
require('dotenv').config();

/**
 * See https://playwright.dev/docs/test-configuration.
 */
export default defineConfig({
    testDir: './tests',
    /* Run tests in files in parallel */
    fullyParallel: true,
    /* Fail the build on CI if you accidentally left test.only in the source code. */
    forbidOnly: !!process.env.CI,
    /* Retry on CI only */
    retries: process.env.CI ? 2 : 0,
    /* Opt out of parallel tests on CI. */
    workers: process.env.CI ? 1 : 8,
    /* Reporter to use. See https://playwright.dev/docs/test-reporters */
    reporter: [['html', {open: process.env.WEB_ENV == 'prod' ? 'on-failure' : 'never'}]],
    /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
    use: {
        // // Emulates the user locale.
        // locale: 'zh-cn',
        // // Emulates the user timezone.
        // timezoneId: 'Asia/Shanghai',
        /* Base URL to use in actions like `await page.goto('/')`. */
        baseURL: process.env.GP_ENV == 'prod' ? 'http:www.xxx.xxx' : process.env.baseUrl ?? 'http://test.xxx.xxx:3000',

        /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
        trace: 'on-first-retry',
        video: 'on-first-retry',
    },
    /* Configure projects for major browsers */
    projects: [
        // {
        //   name: 'chromium',
        //   use: { ...devices['Desktop Chrome'] },
        // },

        // {
        //     name: 'firefox',
        //     use: { ...devices['Desktop Firefox'] },
        // },

        {
          name: 'webkit',
          use: { ...devices['Desktop Safari'] },
        },

        /* Test against mobile viewports. */
        // {
        //   name: 'Mobile Chrome',
        //   use: { ...devices['Pixel 5'] },
        // },
        // {
        //   name: 'Mobile Safari',
        //   use: { ...devices['iPhone 12'] },
        // },

        /* Test against branded browsers. */
        // {
        //   name: 'Microsoft Edge',
        //   use: { ...devices['Desktop Edge'], channel: 'msedge' },
        // },
        // {
        //   name: 'Google Chrome',
        //   use: { ..devices['Desktop Chrome'], channel: 'chrome' },
        // },
    ],

    /* Run your local dev server before starting the tests */
    // webServer: {
    //   command: 'npm run start',
    //   url: 'http://127.0.0.1:3000',
    //   reuseExistingServer: !process.env.CI,
    // },
});
```

**Comment by yury-s** [comment]
Please provide a minimal self-contained project that we could run locally to reproduce the problem.

**Comment by tking16** [comment]
Hi @yury-s, here's the link to the project https://github.com/tking16/playwrightVideoBug

**Comment by yury-s** [comment]
Can you try with latest playwright (1.37.1) and see if the issue is still there? There was a similar bug https://github.com/microsoft/playwright/issues/23839 which is fixed in 1.37

**Comment by yury-s** [comment]
I can confirm that it's the same problem as https://github.com/microsoft/playwright/issues/23839 and is fixed in 1.37.
